resources/sass/components/_global-search.scss

```diff
@@ -0,0 +1,213 @@
+.ris-global-search {
+    $this: &;
+    width: 100%;
+    align-self: center;
+    position: relative;
+    @include breakpoint($bp-mobile-max) {
+        position: static;
+    }
+    &__result-title {
+        padding-left: $indent-ml;
+        margin: $indent-l 0 $indent-m 0;
+        color: $c-gray8;
+        font-size: $fs-l;
+        font-weight: 400;
+    }
+    &-navigation {
+        display: flex;
+        padding-left: $indent-ml;
+        margin: 0;
+        border-bottom: 1px solid $c-gray7;
+        &__button {
+            border: 0;
+            font-size: $fs-l;
+            background-color: transparent;
+            border-bottom: 3px solid transparent;
+            outline: none;
+            &.isActive {
+                color: $c-red;
+                font-weight: bold;
+                border-color: $c-red;
+            }
+        }
+    }
+    &-input {
+        z-index: 7;
+        height: 64px;
+        display: flex;
+        align-items: center;
+        background: white;
+    }
+    &-wrapper {
+        font-size: 10px;
+        background-color: white;
+        position: absolute;
+        left: 0;
+        right: 0;
+        z-index: 6;
+        max-height: calc(100vh - 64px - 48px);
+        overflow-y: scroll;
+        @include breakpoint($bp-mobile-max) {
+            max-height: calc(100vh - 92px);
+            top: 92px;
```

Comment:
if 64, 48 and 92 are values that are somehow related to other elements then they should be variables